crates/ruma-client-api/src/message/get_message_events.rs

```diff
@@ -111,56 +110,40 @@ pub mod v3 {
             }
         }
 
-        /// Creates a new `Request` with the given room ID and optional `from` token, and `dir` set
-        /// to `Backward`.
+        /// Creates a new `Request` with the given room ID and `dir` set to `Backward`.
```

Comment:
Since the other constructors are gone now, I'd probably add in the docs that this retrieves messages from the end of the timeline if `from` is not set.

And something similar for `forward()`.